src/UI/Component/Modal/ReplaceContentSignal.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace ILIAS\UI\Component\Modal;
+
+use ILIAS\UI\Component\Signal;
+
+/**
+ * This signal replaces the content of a Modal with some content returned by an ajax request.
+ *
+ * @author Jesús Lópéz <lopez@leifos.com>
+ * @package ILIAS\UI\Component\Modal
+ */
+interface ReplaceContentSignal extends Signal
```

Comment:
As stated bellow, this is redundant, to what we have for popovers already. This needs to be centralized. However the name does not fit quite as well anymore in this case. Maybe only **ReplaceSignal**?

Reply:
I think ReplaceSignal is better for this case.